### ISSUE TITLE
ansible-lint complains about sshd var naming

### ISSUE THREAD

**Issue by bayramkzk** [willshersystems/ansible-sshd#297]
I am using `ansible-lint` in Neovim and linter complains about `sshd` variable naming convention here:

```yaml
    - name: Configure sshd server daemon config
      ansible.builtin.include_role:
        name: willshersystems.sshd
      vars:
        sshd: # here
          PubkeyAuthentication: true
          PasswordAuthentication: false
          ClientAliveInterval: 60
          Port: "{{ my_sshd_port }}"
        sshd_manage_firewall: true
```

```
site.yaml|27 col 1 error| Variables names from within roles should use sshd_ as a prefix. (vars: sshd)
```

Linter doesn't complain when, say, I rename `sshd` to `sshd_config` but as expected the playbook doesn't work properly then.

**Comment by mattwillsher** [comment]
Hi @bayramkzk 
The [.ansible-lint file excludes this check](https://github.com/willshersystems/ansible-sshd/blob/de66f6065994e79f8caa744766052d503b67c3cb/.ansible-lint#L7). 
There is issue #246 with a view to rename the variables, but that may not be implemented due to the extent of changes required and risks to existing use of the role.

**Comment by mattwillsher** [comment]
I'm going to reopen this as it's different to the issue seen in #246 as this is linting from the calling play rather than the module itself. I can't give time scales on the fix, but [@spetrosi's suggestion ](https://github.com/willshersystems/ansible-sshd/issues/246#issuecomment-2329514112) looks like a good way forward.

**Comment by Jakuje** [comment]
Ugh. It looks like the `sshd` variable itself is an issue for ansible. Not sure how we should name it then. I can think of `sshd_` but I feel that would be just a workaround for this policy and would make it just more confusing for everyone.

The obvious workaround is:
```
      vars:
        sshd_PubkeyAuthentication: true
        sshd_PasswordAuthentication: false
        sshd_ClientAliveInterval: 60
        sshd_Port: "{{ my_sshd_port }}"
```
but I like the first syntax much more.

**Comment by mattwillsher** [comment]
`sshd_config` looks like it's not used. So we could take [this suggestion](https://github.com/willshersystems/ansible-sshd/issues/246#issuecomment-2329514112) to deprecate `sshd`, update the docs etc. to `sshd_config`. 
Or we could use `sshd_sshd`, which looks weird but should stop the linter complaining.

**Comment by mattwillsher** [comment]
I'll pick this up and get a PR put together.

**Comment by Jakuje** [comment]
AFAIK this was fixed with #299. Please, open a new issue if something from this issue was not addressed.
